honeybadger/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.22.1'
```

Comment:
This is automatically modified by the [workflow](https://github.com/honeybadger-io/honeybadger-python/blob/3f50088a57d772f107d3264a865eac4f2021e265/.github/workflows/pypi-publish.yml#L30) file. I think we'll have to modify the workflow to write double quotes instead of single quotes.